neuralmonkey/encoders/cnn_encoder.py

```diff
@@ -172,50 +155,8 @@ def __init__(self, data_id, convolutions, rnn_layers,
                             last_layer, keep_prob=self.dropout_placeholder)
 
                 last_layer = last_layer * last_padding_masks
-            last_layer_size = last_n_channels * image_height * image_width
```

Comment:
Byla za tim cnn ještě optional bidi rnn vrstva a tohle číslo se používalo k reshapování. Tu RNN jsem zahodil, protože mi to přišlo poněkud monster.